deploy/kind/kind-kamaji.yaml

```diff
@@ -14,24 +14,24 @@ nodes:
   extraMounts:
     - hostPath: /var/run/docker.sock
       containerPath: /var/run/docker.sock
-  extraPortMappings:
```

Comment:
This was commented because this extra port mapping is now redundant for E2E testing